Changes

```diff
@@ -227,6 +235,7 @@ Next version (4.05.0):
 - GPR#744, GPR#781: fix duplicate self-reference in imported cmi_crcs
   list in .cmti files + avoid rebuilding cmi_info record when creating
   .cmti files
+<<<<<<< ad60f46f772daec47e5c2c14828eae542353041d
```

Comment:
Whoops. Fixed it. Thank you!
(strangely, magit didn't highlight these lines in diff)